config/routes.rb

```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
-  resources :lists
 
   devise_for :users
-  resources :users, only: [:show]
+  resources :users, only: [:show] do
+    resources :lists do
+      resources :items, only: [:create, :destroy]
```

Comment:
Try to avoid more than 1 level of nesting resources. Here, the item route is nested 2 levels deep, and its URL generation has excessive dependencies as it depends on 3 objects: `user`, `list`, and `item` itself.

Consider removing the `resources :users ...` outer block so `resources :lists ...` is at the top-level in routes.rb.